Assets/Text/cqui_text_settings_pl.xml

```diff
@@ -155,10 +155,10 @@
       <Text>Przełącza wyskakiwanie popup'u który informuje o zakończeniu badania bądź kodeksu prawnego</Text>
     </Row>
     <Row Tag="LOC_CQUI_POPUPS_TECHAUDIO" Language="pl_PL">
-      <Text>Dzwięk zakończenia Badania/Kodeksu</Text>
+      <Text>Odtwarzanie dźwięku za pomocą wyskakujących okienków Tech/Civic</Text>
```

Comment:
The PL translations are plain horrible.
Pls. change into "Odtwarzanie cytatu w okienku ukończenia Technologii/Idei"

Reply:
yeah, it's a roll of the dice with the google translate for any of these.  I'm just hoping they don't end up saying something offensive at this point